### PR TITLE
Pass arguments via env vars at `e2e` tests

### LINKED ISSUE #959: Pass arguments via env vars at `e2e` tests
As a follow up of https://github.com/humanode-network/humanode/pull/633#pullrequestreview-1961934561

### PR DESCRIPTION
**Pull request by dmitrylavrenov**
Closes #959

### CODE REVIEW

File: .github/workflows/code.yml [hunk 1]
```diff
@@ -104,14 +104,18 @@ jobs:
           BUILD_ARGS: ${{ matrix.test.build-args }}
 
       - name: Run bash e2e tests
-        run: utils/e2e-tests/bash/run-tests "$TEST_SUITE" "target/release/humanode-peer" "target/release/wbuild/humanode-runtime/humanode_runtime.wasm"
+        run: utils/e2e-tests/bash/run-tests "$TEST_SUITE"
         env:
           TEST_SUITE: ${{ matrix.test.suite }}
+          HUMANODE_PEER_PATH: "target/release/humanode-peer"
+          RUNTIME_WASM: "target/release/wbuild/humanode-runtime/humanode_runtime.wasm"
+          LIVE_NETWORK_URL: "wss://explorer-rpc-ws.it6.stages.humanode.io:443"
```
Comment (MOZGIII) [inline .github/workflows/code.yml:112]: We should probably run against mainnet

**Review by MOZGIII — APPROVED** [review]

File: utils/e2e-tests/bash/run-tests [hunk 2]
```diff
@@ -6,6 +6,15 @@ cd "$SCRIPT_DIR/../../.."
...
 shift
 
+HUMANODE_PEER_PATH="$(realpath -s "$HUMANODE_PEER_PATH")"
+export HUMANODE_PEER_PATH
+
+RUNTIME_WASM="$(realpath -s "$RUNTIME_WASM")"
+export RUNTIME_WASM
+
+LIVE_NETWORK_URL="$LIVE_NETWORK_URL"
+export LIVE_NETWORK_URL
```
[the remark this answers is not in the document]
Comment (MOZGIII) [reply utils/e2e-tests/bash/run-tests:16]: Wait, why do even need it to be set here?

[the remark this answers is not in the document]
Comment (dmitrylavrenov) [reply utils/e2e-tests/bash/run-tests:16]: I guess it doesn't require at all.

File: .github/workflows/code.yml (hunk 1 above)
Comment (dmitrylavrenov) [reply .github/workflows/code.yml]: https://github.com/humanode-network/humanode/pull/963/commits/bcdb75cd1e306072e39e5f9c13e898517cdef104

File: utils/e2e-tests/bash/run-tests (hunk 2 above)
[the remark this answers is not in the document]
Comment (dmitrylavrenov) [reply utils/e2e-tests/bash/run-tests]: https://github.com/humanode-network/humanode/pull/963/commits/2823d794d61dacceab5d477a0933526d03a23a57

**Pull request merged**